packages/material/scss/scrollview/_theme.scss

```diff
@@ -0,0 +1,34 @@
+$scrollview-navigation-color: $base-text;
+
+@import "~@progress/kendo-theme-default/scss/scrollview/theme";
+
+@include exports("scrollview/theme/material") {
+    $scrollview-text-opacity: .4 !default;
+    $scrollview-bottom: $scrollview-pager-spacing-x * 2;
+
+    .k-scrollview-next,
+    .k-scrollview-prev {
+        .k-icon {
+
+            &:hover {
+
+                &::before {
+                    background-color: rgba($base-bg, $scrollview-text-opacity);
+                }
+            }
+        }
+    }
+
+    .k-scrollview-pageable {
+        bottom: $scrollview-bottom;
+
+        li.k-button {
+            background: rgba(black, $scrollview-text-opacity);
+            margin: $scrollview-pager-spacing-y $scrollview-pager-spacing-x;
+
+            &.k-primary {
+                background: $primary;
+            }
+        }
+    }
+}
```

Comment:
In the default theme, there are styles about this, albeit using different selectors. I guess we could unify the selectors and not add extra styles.

Reply:
@joneff we have different rendering of elements' classes and IMO we should consider make it the same everywhere. @gyoshev what do you think?
These selectors are for overriding the buttons's styles from `theme-material` - `packages\material\scss\buttons\_theme.sccs`, not those from `theme-default`.